lib/util/ast.js

```diff
@@ -4,6 +4,37 @@
 
 'use strict';
 
+const estraverse = require('estraverse');
+
+/**
+ * Wrapper for estraverse.traverse
+ *
+ * @param {ASTNode} ASTnode The AST node being checked
+ * @param {Object} visitor Visitor Object for estraverse
+ */
+function traverse(ASTnode, visitor) {
+  const opts = Object.assign({}, {
+    fallback(node) {
+      return Object.keys(node).filter((key) => {
+        switch (key) {
+          case 'children':
+          case 'argument':
+            return true;
+          default:
+            return false;
+        }
+      });
```

Comment:
switch statements are awful, let's go with this instead:
```suggestion
      return Object.keys(node).filter((key) => key === 'children' || key === 'argument');
```